backend/src/main/java/com/ody/notification/domain/Notification.java

```diff
@@ -48,6 +48,14 @@ public Notification(Mate mate, NotificationType type, LocalDateTime sendAt, Noti
         this(null, mate, type, sendAt, status);
     }
 
+    public static Notification createEntry(Mate mate) {
+        return new Notification(mate, NotificationType.ENTRY, LocalDateTime.now(), NotificationStatus.PENDING);
+    }
+
+    public static Notification createDepartureReminder(Mate mate, LocalDateTime sendAt) {
+        return new Notification(mate, NotificationType.DEPARTURE_REMINDER, sendAt, NotificationStatus.PENDING);
+    }
+
     public void updateStatusToDone() {
```

Comment:
[제안 - 선택]
```suggestion
    public Notification send() {
                     return new Notification(this.mate, this.type, this.sendAt, DONE);
    }
    
```

약간 setter랑 비슷한 느낌이 들어서 그런데 상태 변화를 불변객체로 표현하도록 로직을 수정하면 어떨까 하는 생각이 들었어요. 많은 리소스가 들 수도 있을 것 같아 제안으로만 남겨둡니다.

Reply:
불변을 위해 새로운 객체를 생성하게 되면 
해당 객체를 테이블에 저장하는 작업과 이전 객체를 테이블에서 삭제하는 작업이 추가로 이루어져야 해서
코드가 복잡해진다고 판단해 현재 코드를 유지했습니다!